frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -10,7 +10,7 @@ export interface State {
   genesToDelete: string[];
   selectedGenes: string[];
   selectedOrganismId: string | null;
-  selectedTissues: string[];
+  selectedTissues?: string[];
```

Comment:
These need to be optional since wmg2 will have selectedTissues be empty and v2 endpoint expects no tissues